src/metadataGeneration/typeResolver.ts

```diff
@@ -186,7 +186,12 @@ export class TypeResolver {
 
     const enumDeclaration = enumNodes[0] as ts.EnumDeclaration;
 
+    const typeChecker = this.current.typeChecker;
     function getEnumValue(member: any) {
+      const constantValue = typeChecker.getConstantValue(member);
```

Comment:
Hm, doesn't this return `string | number | undefined`? So shouldn't we check for `undefined` below?